x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/index.tsx

```diff
@@ -74,9 +84,40 @@ export function WaterfallContainer({
   });
 
   return (
-    <div>
-      <WaterfallLegends legends={legendsWithFallbackLabel} type={colorBy} />
-      <Waterfall waterfallItemId={waterfallItemId} waterfall={waterfall} />
-    </div>
+    <EuiFlexGroup direction="column">
+      {isCriticalPathEnabled ? (
```

Comment:
I think the name of the settings and the props are a bit confusing. I think the other way around would make more sense.

Change the settings to `showCriticalPath` and change the props to `enableCriticalPath`.

Reply:
Not sure about swapping them, I think I'll change it to `isCriticalPathFeatureEnabled` or maybe `isCriticalPathFeatureAvailable`. WDYT?

Reply:
I think `isCriticalPathFeatureEnabled` is better.